sessionctx/variable/sysvar.go

```diff
@@ -410,6 +410,18 @@ var defaultSysVars = []*SysVar{
 	}},
 
 	/* The system variables below have GLOBAL scope  */
+	{Scope: ScopeGlobal, Name: TiDBTxnEntrySizeLimit, Value: strconv.FormatInt(DefTiDBTxnEntrySizeLimit, 10), Type: TypeInt, MinValue: 10485760, MaxValue: 125829120, GetGlobal: func(sv *SessionVars) (string, error) {
```

Comment:
Better to use constant min/max-value variables and assign them like `min = 10 << 20, max = 120 << 20`.